pkg/sensors/tracing/kprobe_process_credentials_test.go

```diff
@@ -342,6 +348,11 @@ spec:
 		t.Fatalf("setgid(%d) error: %s", gid+1, err)
 	}
 
+	// Restore all gids to 0
+	if err := syscall.Setgid(0); err != nil {
+		t.Fatalf("setgid(0) error: %s", err)
+	}
```

Comment:
Similar as above: if this is a fix on an existing test, please add it as a separate patch.